utility/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="org.oppia.util">
-  <uses-sdk android:minSdkVersion="19"
+  <uses-sdk
```

Comment:
Note: minSDKversion here was causing Gradle to fail.

Reply:
What was the error? Without additional context, it's hard to know what suggestions to make. We definitely need to specify a minimum otherwise Bazel will assume to use 1 which is incorrect for Oppia Android.

Reply:
I was able to add this back without any issues. See my latest comment below :)